fluentd/configs.d/filter/k8s_record_transform.conf

```diff
@@ -1,4 +1,24 @@
-<filter kubernetes.**>
+<filter kubernetes.journal.container**>
+  type record_transformer
+  enable_ruby
+  <record>
+    hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
+    # log is set if MESSAGE is JSON valued and has err or msg field
+    message ${record['MESSAGE'] || log}
+    version 1.2.0
```

Comment:
we should probably bump all `version` fields up to `1.3.0` since we're introducing some formatting changes (with the introduction of journal)